### PR TITLE
Unresolvable task op should not cause a hard failure

### PR DESCRIPTION
**Pull request by Genza999**
This is an implementation for the CU bug:
https://app.clickup.com/t/14289126/FC-3420

### CODE REVIEW

**Comment by Genza999** [comment]
Task linked: https://app.clickup.com/t/14289126/FC-3420

File: src/fullctl/django/tasks/__init__.py [hunk 1]
```diff
@@ -7,7 +12,12 @@ def register(cls):
 
 
 def specify_task(task):
-    return TASK_MODELS[task.op].objects.get(id=task.id)
+    try:
+        task_model = TASK_MODELS[task.op]
+    except KeyError:
+        log.error("Task operation not found", task_op=task.op)
+        raise
```
Comment (vegu) [inline src/fullctl/django/tasks/__init__.py:19]: @Genza999 i made a typo in the task description - this should NOT actually raise an error, we just want it to log an error and continue on.

**Review by vegu — CHANGES_REQUESTED** [review]

File: src/fullctl/django/tasks/__init__.py (hunk 1 above)
Comment (Genza999) [reply src/fullctl/django/tasks/__init__.py]: This has been refactored, thanks!

**Review by vegu — APPROVED** [review]

**Review by vegu — CHANGES_REQUESTED** [review]
@Genza999 actually - i think this needs another test 

i want us to check that this works smoothly when a task scheduler sets a task op that doesn't exist (which is what caused the creation of the ticket)

so a test that
- sets up a task schedule referencing a non existing task
- runs the fullctl poll task logic
- smoothly identifies that the task doesn't exist, logs an error, and returns to the loop

**Comment by Genza999** [comment]
> @Genza999 actually - i think this needs another test
> 
> i want us to check that this works smoothly when a task scheduler sets a task op that doesn't exist (which is what caused the creation of the ticket)
> 
> so a test that
> 
> * sets up a task schedule referencing a non existing task
> * runs the fullctl poll task logic
> * smoothly identifies that the task doesn't exist, logs an error, and returns to the loop

Good call, turns out this error was being raised from a different point, 
in **create_tasks_from_json** function at 
`task = model.create_task(timeout=timeout, user=user, org=org, *args, **kwargs)`
since if there is no op, there is no model and was causing an Attribute error before continuing processing.

**Comment by vegu** [comment]
@Genza999 needs linting please

**Pull request merged**